jipdate.py

```diff
@@ -229,16 +259,28 @@ def get_jira_issues(jira, username):
 
     f.write(msg)
     vprint("Found issue:")
+
+    unsorted_issues={}
     for issue in my_issues:
+        issue_id = "%s" % issue
+        this_issue = {}
+
+        this_issue["comments"] = ""
         vprint("%s : %s" % (issue, issue.fields.summary))
-        f.write("[%s]\n" % issue)
-        f.write("# Header: %s\n" % issue.fields.summary)
-        f.write("# Type: %s\n" % issue.fields.issuetype)
-        f.write("# Status: %s\n" % issue.fields.status)
-        f.write(get_extra_comments())
-        if last_comment:
-            write_last_jira_comment(f, jira, issue)
-        f.write("\n")
+        this_issue["comments"] += "[%s]\n" % issue
```

Comment:
Could we keep all the writes here as done previously and then follow the same pattern as I did with`write_last_jira_comment`, but instead of an argument, add it as a configuration option in the yaml file? I.e, in the yaml-file:
```yml
show_parent: True
```
And then here (just after line 237, # Status ...) add:
```python
    try:                                                                                      
        if g_yml_config['show_parent']:
             f.write(write_parent(jira, issue))
    except:                                                                                   
        vprint("Parent not found in config")
```

That means that the parent link also would need to go below the issue `ID` itself (and not before as in your patch), for example:
```ini
[SWG-3]
# Header: Generic TrustZone device driver for Linux kernel (upstream)
# Type: Epic
# Status: In Progress
# Parent: SWG-1
# No updates since last week.
```
I think that would make it a bit cleaner and will give people the option to enable/disable parent links at will. Seems like people have lots of different opinions on what should be there by default and how that should look like. In that case it it probably best to make it configurable.